### PR TITLE
Corrected non-linked speaker names

### PR DESCRIPTION
**Pull request by SrishtiG**

### CODE REVIEW

File: content/W2015/W15-event-research-showcase.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ connect with them and get to know more in a mixer setting.
 ## Event Details ##
 
 + **Who:** [Cecylia Bocovich](https://cs.uwaterloo.ca/~cbocovic/), [Erinn Atwater]
-(https://cs.uwaterloo.ca/~aatwater/),[Carolyn Lamb], [Laleh Ghoraie] 
+(https://cs.uwaterloo.ca/~aatwater/),Carolyn Lamb, Laleh Ghoraie
```
Comment (ehashman) [inline content/W2015/W15-event-research-showcase.md]: missing a space before Carolyn's name

**Comment by ehashman** [comment]
I'm going to close this PR and delete this branch; you should not be pushing your personal code to upstream. Please push to your local repo before opening a pull request.

Also, please use descriptive commit messages. It is clear from the patch that you are updating a particular file; the point of the commit message is to say _why_ you are doing so.

**Comment by SrishtiG** [comment]
I'll attempt to re-work this, I'm sorry

**Comment by ehashman** [comment]
It's okay! This was fixed in #41.

**Pull request closed without merging**